### PR TITLE
ADEN-6016 MOAT video tracking countries configurable in WF

### PR DESCRIPTION
**Pull request by fraszczakszymon**

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L78](https://github.com/Wikia/app/blob/384fee5be85eae4890fcf848c0e0616e13aeabfa/extensions/wikia/AdEngine/js/context/adContext.js#L78): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)
1. ![MAJOR][MAJOR] [adContext.js#L86](https://github.com/Wikia/app/blob/384fee5be85eae4890fcf848c0e0616e13aeabfa/extensions/wikia/AdEngine/js/context/adContext.js#L86): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: extensions/wikia/AdEngine/js/context/adContext.js [hunk 1]
```diff
@@ -255,6 +255,7 @@ define('ext.wikia.adEngine.adContext', [
 		);
 
 		context.opts.outstreamVideoFrequencyCapping = instantGlobals.wgAdDriverOutstreamVideoFrequencyCapping;
+		context.opts.porvataMoatTracking = geo.isProperGeo(instantGlobals.wgAdDriverPorvataMoatTrackingCountries);
```
Comment (kvas-damian) [inline extensions/wikia/AdEngine/js/context/adContext.js]: what do you think about `porvataMoatTrackingEnabled`?

**Review by kvas-damian — APPROVED** [review]

File: extensions/wikia/AdEngine/js/context/adContext.js (hunk 1 above)
Comment (fraszczakszymon) [reply extensions/wikia/AdEngine/js/context/adContext.js]: Damn, line will exceed 120 chars :D 
👍 will do

**Review by kvas-damian — APPROVED** [review]

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L78](https://github.com/Wikia/app/blob/e301272b905e111e2dd780003e65a477b30bfebd/extensions/wikia/AdEngine/js/context/adContext.js#L78): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)
1. ![MAJOR][MAJOR] [adContext.js#L86](https://github.com/Wikia/app/blob/e301272b905e111e2dd780003e65a477b30bfebd/extensions/wikia/AdEngine/js/context/adContext.js#L86): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L78](https://github.com/Wikia/app/blob/c7d466a10468dc140b3cc68903bd8f4912886f98/extensions/wikia/AdEngine/js/context/adContext.js#L78): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)
1. ![MAJOR][MAJOR] [adContext.js#L86](https://github.com/Wikia/app/blob/c7d466a10468dc140b3cc68903bd8f4912886f98/extensions/wikia/AdEngine/js/context/adContext.js#L86): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**